src/main/java/Alpaca.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import Tasks.Task;
+
+import Actions.Deadline;
+import Actions.Delete;
+import Actions.Event;
+import Actions.Mark;
+import Actions.ToDo;
+import Actions.Unmark;
+
+import Exceptions.InvalidInput;
+import Exceptions.ValueNotFound;
+
+public class Alpaca {
+    static String name = "Alpaca";
+    static Scanner scanner;
+    static ArrayList<Task> list = new ArrayList<Task>();
+
+    private static void divider() {
```

Comment:
Can we consider using verbs for method names to follow the coding standard guidelines? I noticed the same issue in several other places too.